client/verta/tests/registry/pydantic_models.py

```diff
@@ -0,0 +1,37 @@
+from dataclasses import dataclass
+
+from pydantic import BaseModel
```

Comment:
We should add `pydantic` to https://github.com/VertaAI/modeldb/blob/main/client/verta/requirements-unit-tests.txt